src/controllers/winkels.ts

```diff
@@ -0,0 +1,32 @@
+import { Request, Response } from "express";
+import { getManager, getRepository } from "typeorm";
+import { Store } from "../models/store";
+
+/**
+ * GET /winkels
+ * Shows all stores
+ */
+export async function getStores(req: Request, res: Response): Promise<void> {
+  const stores = await getManager().find(Store);
+  res.render("stores", { stores: stores });
+}
```

Comment:
Ik vind het een beetje vreemd dat we wel winkels.ts hebben, maar dat de functie dan wel "getStores" en "getStore" zijn.

Reply:
Ah, ik snap het al, de winkels is enkel voor de routes, de rest van de code houden we wel engelstalig?

Reply:
Da was min redeneringe idd. Routes zin nederlandstalig en controllers zou ik dezelfde naam geevn als udr routes, al de rest int engels